node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/network/IPVersion.java

```diff
@@ -1,22 +1,27 @@
 package com.yahoo.vespa.hosted.node.admin.task.util.network;
 
 /**
- * Strong type IPv4 and IPv6 with the respective iptables executable.
+ * Strong type IPv4 and IPv6 with common executables for ip related commands.
  *
  * @author smorgrav
  */
 public enum IPVersion {
 
-    IPv6("ip6tables"),
-    IPv4("iptables");
+    IPv6("ip6tables", "ip -6"),
+    IPv4("iptables", "ip");
 
-    IPVersion(String exec) {
-        this.exec = exec;
+    IPVersion(String iptablesCmd, String ipCmd) {
+        this.ipCmd = ipCmd;
+        this.iptablesCmd = iptablesCmd;
     }
 
-    private String exec;
+    private String iptablesCmd;
+    private String ipCmd;
 
-    String exec() {
-        return exec;
+    public String iptablesCmd() {
+        return iptablesCmd;
+    }
+    public String ipCmd() {
```

Comment:
Please avoid acronyms - ipCommand etc.

Reply:
I agree.. but some acronyms are so common that they are more descriptive than the full name. Cmd is one of these (for me at least). A command can be any thing but a cmd has more associations to executables and a terminal environment.